### PR TITLE
Added an Initial-Setup Guide with all the required steps that are required by a contributor to have a local setup running.

### LINKED ISSUE #59: Add a setup guide for local setup.
Currently, there is no local setup guide to setup the sample app up and running. This includes setting up a local ODK Central server as well as setting up `local.properties` and `settings.json`.

### PR DESCRIPTION
**Pull request by prabs3257**
Added an Initial-Setup Guide with all the required steps that are required by a contributor to have a local setup running. Made changes to AndroidManifest.xml of the sample app to ensure that the app trusts the locally added CAs.

Closes #59 

#### What has been done to verify that this works as intended?

Tested and debugged the sample app thoroughly to ensure that no error comes up during the initial local setup

#### Why is this the best possible solution? Were any other approaches considered?

There were other approaches that involved the use of external programs like ngrok to get a public domain. But this solution ensures that the setup depends on minimum external programs. This solution also enhances the local endpoints by making them secure

#### How does this change affect users? Describe intentional changes to behavior and behavior that could have accidentally been affected by code changes. In other words, what are the regression risks?

These changes will not affect the user. Just the contributors who want to have a local setup of the project.

#### Do we need any specific form for testing your changes? If so, please attach one.

NO

#### Does this change require updates to documentation? If so, please file an issue [here]( https://github.com/getodk/docs/issues/new) and include the link below.

I have made the required changes

#### Before submitting this PR, please make sure you have:
- [x] run `./gradlew checkAll` and confirmed all checks still pass OR confirm CircleCI build passes and run `./gradlew connectedDebugAndroidTest` locally.
- [x] verified that any code or assets from external sources are properly credited in comments and/or in the [about file](https://github.com/getodk/collect/blob/master/collect_app/src/main/assets/open_source_licenses.html).
- [x] verified that any new UI elements use theme colors. [UI Components Style guidelines](https://github.com/getodk/collect/blob/master/docs/CODE-GUIDELINES.md#ui-components-style-guidelines)

### CODE REVIEW

File: INITIAL-SETUP.md [hunk 1]
```diff
@@ -0,0 +1,252 @@
...
+    ```
+    git submodule update -i
+    ```
+* Update settings.
+    ```
+    cp .env.template .env
+    nano .env
+    ```
+* Make the following changes in `.env` file
+    * Change the ``Domain`` value to your machine's local IP address. For Eg. DOMAIN=192.168.29.147.
```
Comment (chinmoy12c) [inline INITIAL-SETUP.md:39]: You can check you IP address using `ifconfig`. If you're using ethernet, it should probably be named similar to `eth*`; if you're on WLAN network, it should be `wlo*`.

File: INITIAL-SETUP.md [hunk 2]
```diff
@@ -0,0 +1,252 @@
...
+
+---
+* Now you will need to bundle everyting together. This may take some time
+
+    ```
+    docker compose build
+    ```
+    When it finishes, you should see some "Successfully built" type text 
+
+* Start the server you just created. If its the first time its going to take a while to get ready
```
Comment (chinmoy12c) [inline INITIAL-SETUP.md:106]: Make sure that you do not have any service running on your local port 80.

File: INITIAL-SETUP.md [hunk 3]
```diff
@@ -0,0 +1,252 @@
...
+
+* For the `server-url`.
+    open your server -> create a project -> create a user
+create a form inside the project and go to form access tab
+check the checkbox on your created user
+go to user access tab, you'll see that there is a generate QR code option
+you need to scan this QR on odk collect app (available on playstore)
+once you scan this QR you'll setup a project on odk collect app. Go to settings and then project management. You'll see a server url there
+copy that url and paste it in your settings file. Url should look something like this https://192.169.29.147:443/v1/key/jP5eczJBZq08ECf9OKoNJGwjx55wBa3cpHkJ0r5dPQYEXcXM8uzBOTi38E9NPX$t/projects/2. <br>
+Instead of 192.169.29.147 it should be your local machine's IP
```
Comment (chinmoy12c) [inline INITIAL-SETUP.md]: Please add bullet list to these steps

**Review by chinmoy12c — CHANGES_REQUESTED** [review]
Great work @prabs3257, appreciate the work you've put into this :+1: . Please make the changes that I've pointed out in the PR.

**Comment by prabs3257** [comment]
@chinmoy12c Made the requested changes. Please review it at your earliest convenience.

**Pull request merged**